module/x/gravity/keeper/keeper.go

```diff
@@ -27,7 +27,7 @@ import (
 type Keeper struct {
 	StakingKeeper          types.StakingKeeper
 	EthereumEventProcessor interface {
```

Comment:
this should be defined outside the Keeper struct

Reply:
I just changed it into a keeper method.